exports.login = async(req, res, next) => {
    try{
        // login code here
    } catch(error){
        console.log(error)
    }
}